### ISSUE TITLE
Is it possible to use a named pipe (fifo) to read live audio .wav stream ?

### ISSUE THREAD

**Issue by moving-digital** [naudio/NAudio#907]
Hi, @markheath 

First of all, thank you for this great project.

I'm working on a requirement where I'm transcoding some live audio stream from M1-L2/AAC to .wav, on the fly.

Then, I need to feed NAudio with that live .wav audio stream. 

So I'm wondering if it is possible to use a named pipe (fifo) as input ?

I'm aware this may have its own challenge since ultimately this named pipe is a kinda file, so NAudio may not understand what's going on, therefore my question.

In other words, can NAudio turn a named pipe into an IWaveIn?

If the above is not possible, is there any other solutions?
